Nos/Service/EventProcessor.swift

```diff
@@ -9,26 +9,25 @@ import Foundation
 
 /// The event processor consumes raw event data from the relays and writes it to Core Data.
 enum EventProcessor {
+    static var tempPubKey = ""
```

Comment:
This isn't used anywhere, is it still needed?

Reply:
Good catch. I've removed it.